src/Components/NewTodo.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment, useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTodo } from "services/actions/addTodo";
+
+export default function NewTodo(props) {
+  const { onNewSave } = props;
+  const [todoText, setTodoText] = useState(null);
+
+  const dispatch = useDispatch();
+
+  const formatDate = () => {
+    const today = new Date();
+    const day = String(today.getDate()).padStart(2, "0");
+    const month = String(today.getMonth() + 1).padStart(2, "0"); // Month is zero-based
+    const year = today.getFullYear();
+    return `${day}.${month}.${year}`;
+  };
```

Comment:
Can we move this function is separate helper folder? Which can be use globally by calling the function later. Thanks!

Reply:
moved